lib/WP_Auth0_LoginManager.php

```diff
@@ -333,7 +337,7 @@ public function implicit_login() {
 		// Populate legacy userinfo property.
 		$decoded_token->user_id = $decoded_token->sub;
 
-		if ( $this->login_user( $decoded_token, $token, null ) ) {
+		if ( $this->login_user( $decoded_token, $token ) ) {
```

Comment:
Can the `$_POST['token']` "token" param be renamed before next release? And please rename `$token` to `$id_token` or similar, it's easier to follow up which token is it. 

https://github.com/auth0/wp-auth0/blob/6382a6ca84c1be0ff64a80a4dbfbfc5efeaead39/lib/WP_Auth0_LoginManager.php#L304

Reply:
The key in `$_POST` cannot, unfortunately. That's the name it's had for a while so changing would be breaking. Just changed the internal var name, good call.